### ISSUE TITLE
Solution loading zenroom.wasm through webpack.

### ISSUE THREAD

**Issue by MarcelSchouwenaar** [dyne/Zenroom#81]
### note: this is not a bug or issue in zenroom, but a general recommendation for anyone trying to make zenroom work in the browser with webpack
When using Zenroom in the browser, with a framework like ```Vue.js``` or ```React.js```, .```wasm``` files are parsed by webpack. This produces a cascade of compile errors, like:
``` failed to asynchronously prepare wasm: CompileError: wasm validation error: at offset 0: failed to match magic number ```
and 
```` CompileError: AsyncCompile: Wasm decoding failed: expected magic word 00 61 73 6d, found 3c 21 44 4f @+0````

# Old way breaks with newer webpack
In the following: [guide](https://www.dyne.org/using-zenroom-with-javascript-react-part3/) @puria demonstrated how to deal with this issue, but with more recent versions of webpack this method seems to break.

# Solution:

## webpack config
Following the recommendations in this [guide ](https://wildsilicon.com/blog/2018/emscripten-webpack/) by @dkirkman on how to load  ```.wasm``` files, my webpack config contains the following rule for wasm files:
```javascript
 module: {
    rules: [
      { 
        test: /\.wasm$/,
        type: 'javascript/auto',
        loader: 'file-loader',
        options: {
          name: '[name]-[hash].[ext]',         
        }        
      }
    ]
  },
```
## No need to edit zenroom.js...
TL;DR:
1. leave zenroom.js untouched
2. copy (or symlink) zenroom.wasm to ```public/js```

In the original guide, it was recommended to comment the ```locateFile``` line in ```zenroom.js```. However, webpack seems to search for the file in ```public/js/```. So, keep your ```zenroom.js``` untouched.
```javascript
/* Suppose to comment this in original guide, but: no need to comment this. */
   wasmBinaryFile = locateFile(wasmBinaryFile);
```
## ... and move zenroom.wasm to public/js
and move your zenroom.wasm to public/js

**Comment by jaromil** [comment]
Many thanks Marcel! @puria_dyne what is the status of the wasm builds? should we simply include this issue in the documentation?
